internal/testrunner/runners/system/servicedeployer/custom_agent.go

```diff
@@ -154,5 +154,5 @@ func (d *CustomAgentDeployer) loadComposeDefinitions() ([]string, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "can't locate Docker Compose file for Custom Agent deployer")
 	}
-	return []string{locationManager.DockerCustomAgentDeployerYml(), d.cfg}, nil
+	return []string{d.cfg,locationManager.DockerCustomAgentDeployerYml()}, nil
```

Comment:
Nit. The linter is complaining I think that because of the lack of space between elements.
```suggestion
	return []string{d.cfg, locationManager.DockerCustomAgentDeployerYml()}, nil
```